### PR TITLE
Docs and cleanup

### PR DESCRIPTION
**Pull request by mds1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,3 +99,12 @@ If this project needs something other than the vanilla commands like `forge test
 ## Other stuff
```
Comment (apbendi) [inline README.md:99]: What's the point of this "Other Stuff" section?

File: README.md [hunk 2]
```diff
@@ -2,27 +2,93 @@
...
+However, you can [configure](https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/configuring-code-scanning#defining-the-severities-causing-pull-request-check-failure) which level of slither results cause PR check failures.
+
+It's recommended to conservatively set the failure level to `Any` to start, and to reduce the failure level if you are unable to sufficiently tune Slither or find it to be too noisy.
+
+Findings are shown directly on the PR, as well as in your repo's "Security" tab, under the "Code scanning" section.
+Alerts that are dismissed are remembered by GitHub, and will not be shown again on future PRs.
+
+Note that code scanning integration [only works](https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/setting-up-code-scanning-for-a-repository) for public repos, or private repos with GitHub Enterprise Cloud and a license for GitHub Advanced Security.
```
Comment (apbendi) [inline README.md:91]: This is pretty pricey, right? Is there a recommended alternative configuration that still includes Slither output for a closed repo in some way w/o having this service level from GitHub? Could definitely imagine someone wanting to use the template for an under-development project that wasn't yet open sourced.

File: README.md (hunk 1 above)
Comment (mds1) [reply README.md:99]: Heh, originally the README was more bare so it felt more like a template readme with some defaults you can keep. This included:
- The (still present) "If this project needs something other than the vanilla commands like `forge test`, or needs environment variables or RPC URLs, mention that here." text.
- The "other stuff, do whatever else you need" section

I'm thinking we remove both and make the README purely informational about the template?

File: README.md (hunk 2 above)
Comment (mds1) [reply README.md:91]: Looks like Enterprise pricing is $252/user/year, but Advanced Security is an add on and I can't find how much it costs, even on the [billing help](https://docs.github.com/en/billing/managing-billing-for-github-advanced-security/about-billing-for-github-advanced-security) page.

IMO the workaround would be to remove the code scanning integration and use slither normally. It has a triage mode which behaves similarly, e.g. "mark this finding as ignored and we won't report it again". So I think you'd probably be able to use that and commit the triage results. Figured I'd figure that one out more when we have the need, but maybe worth mentioning the workaround?

Comment (mds1) [reply README.md:99]: Updated in 09cf570

File: README.md (hunk 1 above)
Comment (mds1) [reply README.md]: Updated in 09cf570

**Review by apbendi — APPROVED** [review]

**Pull request merged**